fuzzers/frida_gdiplus/src/fuzzer.rs

```diff
@@ -86,7 +86,7 @@ unsafe fn fuzz(options: &FuzzerOptions) -> Result<(), Error> {
             unsafe extern "C" fn(data: *const u8, size: usize) -> i32,
         > = lib.get(options.harness_function.as_bytes()).unwrap();
 
-        let mut frida_harness = |input: &BytesInput| {
+        let mut frida_harness = |input: &BytesInput, _executor_state: &mut ()| {
```

Comment:
I'd add a new method to executors that's `::with_state` or something, and takes this new harness (since most of the time we don't care)
Alternatively, we could implement `From<FnMut<...>>` that adapts the closures, maybe?